examples/java/pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.beam</groupId>
     <artifactId>beam-examples-parent</artifactId>
-    <version>0.2.0-incubating-SNAPSHOT</version>
```

Comment:
I think you want to revert the pom files which you edited to validate the change.